components/snackbar/snackbar.css

```diff
@@ -0,0 +1,98 @@
+.show-snackbar-baseline{
+    visibility: hidden;
+    position: fixed;
+    bottom: 2rem;
+    z-index: 2;
+    max-width: 500px;
+    width: 100%;
+    display: flex;
+    flex-flow: row wrap;
+    background-color: var(--snackbarbg);
+    justify-content: space-between;
+    align-items: center;
+    margin: 2rem;
+    border-radius: 0.5rem;
+}
+.show-snackbar-leading{
+    visibility: hidden;
+    position: fixed;
+    bottom: 2rem;
+    z-index: 2;
+    max-width: 500px;
+    width: 100%;
+    display: flex;
+    flex-flow: row wrap;
+    background-color: var(--snackbarbg);
+    justify-content: space-between;
+    align-items: center;
+    margin: 2rem;
+    border-radius: 0.5rem;
+}
+
+.snackbar-left-comp{
+    padding: 1rem;
+    color: white;
```

Comment:
you can assign a variable to white color too, as a good practice

Reply:
Sure will refactor that.